src/ophyd_async/epics/motor.py

```diff
@@ -228,11 +238,18 @@ async def _prepare_velocity(
 
     async def locate(self) -> Location[float]:
         """Return the current setpoint and readback of the motor."""
-        location: Location = {
-            "setpoint": await self.user_setpoint.get_value(),
-            "readback": await self.user_readback.get_value(),
-        }
-        return location
+        setpoint, readback = await asyncio.gather(
+            self.user_setpoint.get_value(), self.user_readback.get_value()
+        )
+        return Location(setpoint=setpoint, readback=readback)
+
+    def subscribe(self, function: Callback[dict[str, Reading[float]]]) -> None:
+        """Subscribe."""
+        self.user_readback.subscribe(function)
+
+    def clear_sub(self, function: Callback[dict[str, Reading[float]]]) -> None:
+        """Unsubscribe."""
+        self.user_readback.clear_sub(function)
```

Comment:
Could: Might want a unit test covering just these motor changes?

Reply:
This is temporary until https://github.com/bluesky/ophyd-async/issues/828 so I'll cover tests as part of that ticket